### DISCUSSION TITLE
Boot from Hard Drive?

### DISCUSSION THREAD

**Discussion by cdaters** [dirkwhoffmann/vAmiga#768] [Q&A] [answered]
Is there a trick or secret to installing Workbench 3.1 to a hard drive? In an Amiga 1200 machine I am setting up in vAmiga,  I have created a drive image using both OFS and FFS using different Workbench install media several times. The installation goes off without a hitch each time. When it gets to the point of ejecting the installation media before clicking on the "Proceed" button to reboot, the emulated machine just gets stuck at the "insert disk" animation after it reboots.

I've seen the "[Classic Workbenches](http://classicwb.abime.net/index.htm) in HDF format" link in the wiki, but I would like to try and install this myself for the nostalgic "feel" of the process.

**Comment by nicolasbauw** [comment]
I had the same issue, when my drive parameters were saved (I thought...) in the machine config file.
To make it work, I had to enable the HDD controllers in the peripherals panel, and set them as default by clicking on "Use as default" (and not use a config file at all).
<img width="641" alt="image" src="https://user-images.githubusercontent.com/37193744/208448384-2e84b63a-50fe-445c-b0df-176b80f536c3.png">

**Comment by cdaters** [comment]
So I have only gotten this to work once. Is there a particular order to "Use as Default" when trying to install Workbench onto a hard drive?

I've configured my A1200, and set the preferences accordingly... I "Use as Default" after creating my 128MB FFS AmigaHD (exporting it as a folder on my machine), and save my A1200.vAmiga setup. Then I proceed to install Workbench (trying it one last time as we speak)... When this is done, I then "Save" my machine setup one last time before I click that "Proceed" button to reboot and cross my fingers that it will boot from the HD I setup and just finished installing Workbench to...

**Comment by dirkwhoffmann** [comment, marked as answer]
After installing, you should export your hard drive as an HDF (via the HDF menu). After restarting vAmiga, simply drag and drop the HDF into the emulator window. "Use as default" will remember the emulator settings (such as if a Zorro HD controller is present), but it won't save your hard drive contents. 

vAmiga's design principle is to *not* modify any ADF or HDF on the host system. If you want to preserve changes made inside the virtual Amiga, you need to export the media (ADF, HDF) explicitly before quitting the app.

**Reply by cdaters** [reply]
Thank you very much Dirk!
